application.js

```diff
@@ -36,7 +41,7 @@ function resetDataset(req, res) {
   // Define titles for default, updates and collision collections in MongoDB.
   const dataset = req.params.datasetId;
   const updates = dataset + '-updates';
-  const collisions = dataset + '-collision';
+  const collisions = dataset + '_collision';
```

Comment:
It turns out the naming conventions for the updates and collisions collections differ. This will now empty the collision collection too.